src/completion-provider.js

```diff
@@ -75,18 +83,15 @@ class CompletionProvider {
     return filename.replace(/\.(js|es6|jsx|coffee)$/, '');
   }
 
-  lookupGlobal(prefix) {
-    const projectPath = atom.project.getPaths()[0];
-    if (!projectPath) {
-      return [];
-    }
+  lookupGlobal(prefix, vendor = 'node_modules') {
+    const projectPath = atom.project.getPaths()[0] || [];
```

Comment:
Upon further reflection, I need to add the early return back. I removed it in one of my iterations of the solution and never added it back.